snapshot/filer.go

```diff
@@ -6,8 +6,7 @@ package snapshot
 // A Filer lets clients deal with Snapshots as files in the local filesystem.
 type Filer interface {
 	Checkouter
-	// TODO(dbentley): support ingestion
-	// Ingester
+	Ingester
```

Comment:
FWIW, the English word is Ingestor, but the Go style is with an -er, so I'm fine with either.